frontend-streamhub-main/src/components/Navbar.tsx

```diff
@@ -25,12 +24,11 @@ const Navbar = () => {
             <img src={logo} alt="StreamHub Logo" />
           </Link>
         </div>
-        <TestApiButton />
 
         <NavigationMenuList className="px-2 py-4 flex justify-center">
           <NavigationMenuItem>
             <Button variant="ghost" className={buttonTextFormat} asChild>
-              <Link to={`/watch-party/1456`}>Join a Watch Party</Link>
+              <Link to={`/watch-party/1456`}>Create a Watch Party</Link>
             </Button>
           </NavigationMenuItem>
```

Comment:
Add margin / spacing between buttons because the buttons are too close when you hover over it

Reply:
yes ma'am